.travis/dictionaries/ro.pws

```diff
@@ -0,0 +1,2175 @@
+personal_ws-1.1 ro 0 utf-8
```

Comment:
The Romanian dictionary is missing *a lot* of words. @Matei-Gavris: could you confirm that the words in this file are correct Romanian words please?